shell/conf/benchmarks_defaults.conf

```diff
@@ -85,7 +85,7 @@ BENCH_SCALE_FACTOR="$(( BENCH_DATA_SIZE / 1000000000 ))" #in GB
 [ ! "$HIVE_FILEFORMAT" ] && HIVE_FILEFORMAT="ORC" # Available options are: EXTFILE, RCFILE, ORC(default), SEQUENCEFILE, PARQUET, AVRO,
 [ ! "$HIVE_ENGINE" ] && HIVE_ENGINE="tez" # Available options are: tez (default, hadoop 2 only), mr (MapReduce)
 [ ! "$HIVE_ML_FRAMEWORK" ] && HIVE_ML_FRAMEWORK="mahout" # Available options are: spark, spark-2, mahout(default)
-[ ! "$BB_SERVER_DERBY" ] && BB_SERVER_DERBY="true" # Available options are: true(Server/Client deployment), false(embeded only)
+[ ! "$BB_SERVER_DERBY" ] && BB_SERVER_DERBY=1 # Available options are: true(Server/Client deployment), false(embeded only)
```

Comment:
Comment should be updated to match the new behaviour. Do/did we ever check for `false` before? To be honest, this one might make more sense with descriptive strings, e.g. `embedded` instead of `false` (or change the variable name to something a bit more descriptive, like `$BB_DERBY_USE_EMBEDDED`).

....or just don't touch it at all in this commit!

Reply:
Many Aloja variables need a refactor, for instance ENGINE={hive,spark-sql} needs to be renamed. I propose to do a refactor of benchmark_defaults and change all related variables in a  single branch. We also need to discuss standardise the name and the way to populate the variables in different conditions; for instance when using a control variable, should use_spark be set to 1 or to "true", etc.

Reply:
I'm not proposing that we do sweeping refactors through the whole code base to change these things, I'm just saying that if you change the code here, you should update the comment to match. 

For now I'm happy with using `1` and `0` for the new variables (and making sure the tests check the value, rather than just whether the variable is set or not).

The thing about `ENGINE` seems completely unrelated (and I don't understand what the problem with the current names is).